scripts/helper.js

```diff
@@ -706,25 +706,27 @@ HELPER.generate = (() => {
 			// iterate over all packages
 			if( allModules !== undefined && allModules.length > 0 ) {
 				for( let module of allModules ) {
-					const pkg = require( Path.normalize(`${ __dirname }/../packages/${ module }/package.json`) );
-					let jquery = '';
-					let react = '';
-
-					if( pkg.pancake['pancake-module'].jquery ) {
-						jquery = `<a class="link" href="packages/${ module }/tests/jquery/">jquery</a>`;
-					}
-
-					if( pkg.pancake['pancake-module'].react ) {
-						react = `<a class="link" href="packages/${ module }/tests/react/">react</a>`;
+					const pkg = require(Path.normalize(`${__dirname}/../packages/${module}/package.json`));
+					if (pkg.name.indexOf('ngx') === -1) {
```

Comment:
I'm not convinced name spacing is the best way to id packages... We have used config options in each package.json file before.

Reply:
Yeah .. that one .. I had to find a way not to make an Angular library look like a React library. That was a quick and dirty solution :).